web_monitoring/diffing_server.py

```diff
@@ -233,7 +234,7 @@ class BoundDiffHandler(DiffHandler):
     return tornado.web.Application([
         (r"/([A-Za-z0-9_]+)", BoundDiffHandler),
         (r"/", IndexHandler),
-    ])
+    ], debug=os.environ.get('TORNADO_DEBUG_MODE', 'False'))
```

Comment:
I think it’s a bad idea to read the environment here—instead, `make_app()` should take debug mode as an argument and you should read the value from the environment either in `cli()` or read it into a constant at the top of the file.

It’s a small thing, but that helps:

1. Keep environment reading confined to a single place
2. Make it possible to always instantiate a copy of the server in code with different settings from whatever’s global.

Also, does `'False'` actually work correctly here? I would have expected a non-empty string to be truthy, since this is supposed to take a boolean value.

Reply:
Moved it to the beginning of the file and I am now checking better to see if it is true or false. Thank you!